crates/bevy_ecs/src/system/system_registry.rs

```diff
@@ -74,6 +74,16 @@ impl<I, O> std::fmt::Debug for SystemId<I, O> {
     }
 }
 
+impl<I, O> From<SystemId<I, O>> for Entity {
+    /// It's trivial to convert [`SystemId`] into an [`Entity`] since a system
```

Comment:
```suggestion
     /// Transforms a [`SystemId`] into the [`Entity`] that holds the one-shot system's state.
     ///
    /// It's trivial to convert [`SystemId`] into an [`Entity`] since a system
```